src/components/constants.ts

```diff
@@ -316,8 +316,20 @@ export const CHECK_LIST_SORT_OPTIONS = [
     value: CheckSort.ZToA,
   },
   {
-    label: 'Success',
-    value: CheckSort.SuccessRate,
+    label: 'Asc. Reachability ',
```

Comment:
I added these whilst refactoring as I thought they made a lot of sense. Should I add latency Asc / Desc, too?